IdentityCore/src/cache/token/MSIDAccountCacheItem.m

```diff
@@ -32,6 +32,11 @@ @interface MSIDAccountCacheItem()
 
 @implementation MSIDAccountCacheItem
 
+- (NSString *)description
```

Comment:
nit: do we want to hide PII account identifiers?

Reply:
We should print the description of MSIDAccountCacheItem only if PII enabled.